powered-by.html

```diff
@@ -2,453 +2,452 @@
-            "description": "Rabobank is one of the 3 largest banks in the Netherlands. Its digital nervous system, the Business Event Bus, is powered by Apache Kafka. It is used by an increasing amount of financial processes and services, one which is Rabo Alerts. This service alerts customers in real-time upon financial events and is built using Kafka Streams."
-        },{
-            "link":  "http://www.portoseguro.com.br/",
-            "logo": "porto-seguro.png",
-            "logoBgColor": "#ffffff",
-            "description": "We use Kafka in production for online and near real-time solutions. Kafka is a core part for many products, such as our Credit Card System."
-        },{
-            "link":  "https://empathy.micronauticsresearch.com/",
-            "logo": "robotCircle.png",
-            "logoBgColor": "#ffffff",
-            "description": "<a href 'https://empathy.micronauticsresearch.com/' target=blank'> EmpathyWorks</a> is a framework for simulating and analyzing networks of artificial personalities."
-        },{
-            "link":  "https://www.cj.com/",
-            "logo": "CJ_Affiliate.png",
-            "logoBgColor": "#ffffff",
-            "description": "Apache Kafka is used at CJ Affiliate to process many of the key events driving our core product. Nearly every aspect of CJ's products and services currently benefit from the speed and stability this provides; additionally, Apache Kafka is one of the key technologies enabling CJ's upcoming real-time Insights & Analytics platform."
-        }, {
-            "link":  "http://xitenetworks.com/",
-            "logo": "xite.png",
-            "logoBgColor": "#ffffff",
-            "description": "Kafka is at the heart of our Data Infrastructure - Business Intelligence, Recommender Systems and Machine Learning solutions are build as reactive and streaming architecture. Also we use Kafka as a great alternative to REST APIs for micro-services integration. This allows us to scale and reliably upgrade micro-services without integration and consistency issues."
-        },{
-            "link":  "http://yahoo.com",
-            "logo": "yahoo.png",
-            "logoBgColor": "#3d018b",
-            "description": "See <a href='http://yahooeng.tumblr.com/post/109994930921/kafka-yahoo' target='_blank'>this</a>."
-        }, {
-            "link":  "https://linecorp.com/",
-            "logo": "line.png",
-            "logoBgColor": "#00b900",
-            "description": "LINE uses Apache Kafka as a central datahub for our services to communicate to one another. Hundreds of billions of messages are produced daily and are used to execute various business logic, threat detection, search indexing and data analysis. LINE leverages Kafka Streams to reliably transform and filter topics enabling sub topics consumers can efficiently consume, meanwhile retaining easy maintainability thanks to its sophisticated yet minimal code base."
-        }
-    ];
+    {
```

Comment:
Just a meta comment: re-arranging items and re-formatting at the same time would make the diff file very hard to review.

I'd assume there is no content changes in this file, otherwise please lmk @manjuapu .

Reply:
@guozhangwang no content change.

Reply:
@joel-hamill I have removed the s from Developers. And left the nav bar as it is now as per our discussion.